### DISCUSSION TITLE
v6.8.0

### DISCUSSION THREAD

**Discussion by ssbarnea** [ansible/ansible-lint#2568] [General]
## Minor Changes

- Add no-shorthand rule as experimental (#2527) @ssbarnea
- Adds summary stats about found violations (#2495) @ssbarnea
- Upgrade fqcn-builtins rule into fqcn (#2505) @ssbarnea
- Changed success/failure message (#2567) @ssbarnea

## Bugfixes

- Add extra documentation (#2503 #2511 #2546 #2551 #2552 #2554 #2555 #2559 #2560 #2561) @ssbarnea @oraNod
- Refactor the rendering of errors and warnings (#2566) @ssbarnea
- Fixed regex for pipefail rule (#2564) @ajinkyau
- Ensure we detect templating errors (#2558) @ssbarnea
- Remove enrich as a dependency (#2557) @ssbarnea
- Implement strict mode (#2533) @ssbarnea
- Avoid extra space when building task names (#2531) @ssbarnea
- Refactor parsing of ansible syntax check (#2529) @ssbarnea
- Allow var_files to be null, string and list of strings (#2528) @ssbarnea
- Fix bug which prevented run of rules on tasks doing imports (#2526) @ssbarnea
- Resolved documentation redirects (#2523) @ssbarnea
- Document lack of module_defaults support (#2519) @ssbarnea
- Allow file attribute with ansible.builtin.include_tasks (#2518) @ssbarnea
- Update schemas (#2517) @ssbarnea
- Replace template-instead-of-copy with avoid-implicit[copy-content] (#2512) @ssbarnea
- Fix path expansion when outside user home (#2515) @ssbarnea
- var-naming: also recognize vars with set_fact module (#2496) @nishipy
- Fix rich exception with tasks containing square brackets (#2510) @ssbarnea
- Fix exception with vars_prompt (#2500) @ssbarnea
- Fix project_dir detect with .config configs (#2499) @ssbarnea
- Avoid implicit templating if not play/task (#2498) @ssbarnea
- Fixed few profile rule name errors (#2497) @ssbarnea
- Rename no-shorthand to no-free-form (#2548) @ssbarnea
- Avoid reporting matches for files outside project_dir (#2547) @ssbarnea
- Allow fqcn with more than 2 dots (#2544) @ssbarnea
- no-shorthand: Avoid false positive with raw (#2541 #2542) @ssbarnea
- Fix project directory detection with config inside .config (#2540) @ssbarnea

<hr /><em>This discussion was created from the release <a href='https://github.com/ansible/ansible-lint/releases/tag/v6.8.0'>v6.8.0</a>.</em>

**Comment by ssbarnea** [comment]
If anyone is courios about how the summary looks, here is an example
![](https://sbarnea.com/ss/Screen-Shot-2022-10-06-20-16-45.59.png)

**Comment by ziegenberg** [comment]
This is awesome! Thanks for the hard work 🙏🏻
